scylla-artifacts.py

```diff
@@ -495,6 +495,31 @@ def enhanced_net_enabled(self):
         else:
             self.log.info("The instance (%s) doesn't support enahanced networking!", result.stdout)
 
+        result = process.run('scylla --version')
+        ver = re.findall("(\d+.\d+)", result.stdout)[0]
+        request_ver = '2017.666' if self.is_enterprise else '2.0'
+        if parse_version(ver) < parse_version(request_ver) and maintype == 'i3':
+            result = process.run('cat /etc/scylla.d/io.conf |grep -v \#',
+                                 shell=True,
+                                 ignore_status=True,
+                                 verbose=True)
+            assert result.stdout != ''
+            result = process.run('cat /etc/scylla.d/cpuset.conf |grep -v \#',
+                                 shell=True,
+                                 ignore_status=True,
+                                 verbose=True)
+            assert result.stdout != ''
```

Comment:
The only comment here is that you could do that in python, instead of resorting to the shell one liner.